MessengerProj/src/main/res/values/strings.xml

```diff
@@ -220,7 +220,7 @@
     <string name="Intro4Message"><![CDATA[<b>Unlimited</b>]]> chats, images, videos, audio messages and more. Multi-client capable.</string>
 
     <string name="Intro5Headline">Free</string>
-    <string name="Intro5Message"><![CDATA[<b>Delta Chat</b>]]> is free forever.<![CDATA[<br/>]]>Open-source. No ads. No subscription. No vendor lock-in.</string>
+    <string name="Intro5Message"><![CDATA[<b>Delta Chat</b>]]> is forever gratis.<![CDATA[<br/>]]>Copylefted libre software. No ads. No subscription. No vendor lock-in.</string>
```

Comment:
same here, i would leave this sentence untouched, this would require a broader discussion:
the "free" here means more than "gratis" here (maybe the icon is not correct).
Also "Open-source" is more known than "Copylefted libre".

Reply:
Which is when it gets localized into, and equivalents of "gratis", that duality is unambiguous in how it only translates into price.

Reply:
yes, but i have never meant only the price here.
it's free in many senses, including price, but not limited to that.
might be that translators will translate if to "gratis", which would be a wrong translation then, however, it's wrong in _one_ locale then - if we change the meaning of the english source, _all_ locales will have this error.
but we may add translator hint above the line.

as mentioned, the wording itself i would leave untouched for the moment.